src/slacker/client/cluster.clj

```diff
@@ -82,43 +82,49 @@
 (defn ^:no-doc group-call-results [grouping-results
                                    grouping-exceptions
                                    call-results]
-  (let [req-info (dissoc (first call-results) :result :cause)]
-    (doseq [r (filter :cause call-results)]
-      (logging/info (str "error calling "
+  (let [req-info (dissoc (first call-results) :result :cause)
+        failed-results (filter :cause call-results)]
+    (doseq [r failed-results]
+      (logging/warn (str "error calling "
                          (:server r)
                          ". Error: "
                          (:cause r))))
     (cond
```

Comment:
Perhaps if we allow user to define a `grouping-exception` function, we should check it at very first priority inside `cond`. Because the `grouping-exception` may want to ignore any exception and see everything as success call.

Reply:
👍 Please wait a moment, I'll try to fix it